net/udpspeeder/Makefile

```diff
@@ -46,7 +47,7 @@ define Build/Prepare
 	$(Build/Patch)
 endef
 
-define Package/udpspeeder/install
+define Package/UDPspeeder/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/speederv2_cross $(1)/usr/bin/udpspeeder
```

Comment:
I'd use PKG_INSTALL with PKG_INSTALL_DIR.

Reply:
This software doesn't have any configure script and builds the single binary in the build directory itself. That's why I have to use PKG_BUILD_DIR here.

Reply:
Then why do you have autoreconf above?

Reply:
I had picked the Makefile from some other source and modified it. This is my first time sending pull request for adding a  completely new package.

Should I convert it to a draft?

Reply:
Removed autoreconf. Please review when you have time for further changes required.

One thing that I am unable to do is run multiple named instances as it happens with OpenVPN. As of now, we can start / stop / restart all the instances at once but not selectively. I will work on the init script to be able to do that soon.